mlflow/utils/search_utils.py

```diff
@@ -819,3 +819,69 @@ def parse_filter_for_registered_models(cls, filter_string):
         return cls._parse_filter_for_model_registry(
             filter_string, cls.VALID_SEARCH_KEYS_FOR_REGISTERED_MODELS
         )
+
+
+class SearchExperimentsUtils(SearchUtils):
+    VALID_SEARCH_ATTRIBUTE_KEYS = ("name",)
+    VALID_ORDER_BY_ATTRIBUTE_KEYS = ("name",)
```

Comment:
We should also support ordering by ID and creation time

Reply:
Can we add tests for these too?

Reply:
> We should also support ordering by ID and creation time

In OSS MLflow, the experiments table doesn't have a `creation_time` column.

https://github.com/mlflow/mlflow/blob/8693a26bf2acd418aa93aa5ea2e5536caa199fc3/tests/db/schemas/sqlite.sql#L8-L15

Reply:
Experiment ID can be supported. I'll add it.

Reply:
Could we add a creation_time column?